iris_grib/_load_convert.py

```diff
@@ -776,10 +776,35 @@ def grid_definition_template_10(section, metadata):
     # intersects the Earth
     standard_parallel = section["LaD"] * _GRID_ACCURACY_IN_DEGREES
 
+    if "orientationOfTheGrid" in section and not np.isclose(
```

Comment:
What's the need for "if ... in section" here -- have I misunderstood something ?
By my understanding, if gdt is 10 then this key _should_ always be present, though it may have missing/None value ?